<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
    ~~~~~~~~~~~~~~~~^
        recipe,
        ^^^^^^^
        fix=fix,
        ^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
    ~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/check_syntax.py", line 36, in check_recipe
    has_constraints = constraints.search(spec)
TypeError: expected string or bytes-like object, got 'NoneType'

ERROR conda.cli.main_run:execute(127): `conda run conda-lint /tmp/abs_89jlnc_k1r/clone` failed. (See above for error)
</details>

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -8,7 +8,14 @@ mkdir -p ${PREFIX}/lib/stubs
 
 [[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
 [[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
-[[ ${target_platform} == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+# https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html?highlight=tegra#cross-compilation
+[[ ${target_platform} == "linux-aarch64" && ${arm_variant_type} == "sbsa" ]] && targetsDir="targets/sbsa-linux"
+[[ ${target_platform} == "linux-aarch64" && ${arm_variant_type} == "tegra" ]] && targetsDir="targets/aarch64-linux"
+
+if [ -z "${targetsDir+x}" ]; then
+    echo "target_platform: ${target_platform} is unknown! targetsDir must be defined!" >&2
+    exit 1
+fi
 
 for i in `ls`; do
     [[ $i == "build_env_setup.sh" ]] && continue
@@ -35,3 +42,5 @@ for i in `ls`; do
         cp -rv $i ${PREFIX}/${targetsDir}/${PKG_NAME}/$i
     fi
 done
+
+check-glibc ${PREFIX}/${targetsDir}/lib/*.so.*

--- a/recipe/conda_build_config.yaml
+++ b/recipe/conda_build_config.yaml
@@ -1,5 +1,3 @@
 arm_variant_type: # [aarch64]
   - sbsa          # [aarch64]
-conda_glibc_ver:
-  - 2.17          # [not aarch64]
-  - 2.26          # [aarch64]
+  - tegra         # [aarch64]

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -94,6 +94,9 @@ outputs:
     test:
       requires:
         - patchelf  # [linux]
+        # We need objdump on linux from binutils:
+        # test-rpath.sh: line 16: objdump: command not found
+        - binutils  # [linux]
       files:
         - test-rpath.sh
       commands:

--- a/recipe/test-rpath.sh
+++ b/recipe/test-rpath.sh
@@ -2,7 +2,14 @@
 
 [[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
 [[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
-[[ ${target_platform} == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+# https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html?highlight=tegra#cross-compilation
+[[ ${target_platform} == "linux-aarch64" && ${arm_variant_type} == "sbsa" ]] && targetsDir="targets/sbsa-linux"
+[[ ${target_platform} == "linux-aarch64" && ${arm_variant_type} == "tegra" ]] && targetsDir="targets/aarch64-linux"
+
+if [ -z "${targetsDir+x}" ]; then
+    echo "target_platform: ${target_platform} is unknown! targetsDir must be defined!" >&2
+    exit 1
+fi
 
 errors=""
 

--- a/recipe/test-rpath.sh
+++ b/recipe/test-rpath.sh
@@ -13,7 +13,7 @@ for lib in `find ${PREFIX}/${targetsDir}/lib -type f`; do
     echo "$lib rpath: $rpath"
     if [[ $rpath != "\$ORIGIN" ]]; then
         errors+="$lib\n"
-    elif [[ $(objdump -x ${lib} | grep "PATH") == *"RUNPATH"* ]]; then
+    elif command -v objdump >/dev/null 2>&1 && [[ $(objdump -x ${lib} | grep "PATH") == *"RUNPATH"* ]]; then
         errors+="$lib\n"
     fi
 done

